py/desispec/qa/qa_exposure.py

```diff
@@ -0,0 +1,76 @@
+"""
+Class to organize and execute QA for a DESI exposure
+"""
+
+from __future__ import print_function, absolute_import, division, unicode_literals
+
+import numpy as np
+
+class QA_Exposure(object):
+    def __init__(self, exptype='none', expid=-1, in_data=None):
```

Comment:
Following SDSS/BOSS, we've been using "flavor" instead of "exptype" elsewhere within desi code and files.